angular.json

```diff
@@ -779,6 +779,111 @@
                     "defaultConfiguration": "development"
                 }
             }
+        },
+        "manager": {
+            "projectType": "application",
+            "schematics": {
+                "@schematics/angular:component": {
+                    "style": "scss",
+                    "standalone": false
+                },
+                "@schematics/angular:directive": {
+                    "standalone": false
+                },
+                "@schematics/angular:pipe": {
+                    "standalone": false
+                }
+            },
+            "root": "projects/manager",
+            "sourceRoot": "projects/manager/src",
+            "prefix": "app",
+            "architect": {
+                "build": {
+                    "builder": "@angular-devkit/build-angular:application",
+                    "options": {
+                        "outputPath": "dist/manager",
+                        "index": "projects/manager/src/index.html",
+                        "browser": "projects/manager/src/main.ts",
+                        "polyfills": ["zone.js"],
+                        "tsConfig": "projects/manager/tsconfig.app.json",
+                        "inlineStyleLanguage": "scss",
+                        "assets": [
+                            "projects/manager/src/favicon.ico",
+                            "projects/manager/src/assets",
+                            {
+                                "glob": "**/*",
+                                "input": "dist/core/assets",
+                                "output": "assets"
+                            }
+                        ],
+                        "styles": [
+                            "projects/manager/src/styles.scss",
+                            "dist/core/assets/fonts/poppins/poppins.css",
+                            "dist/core/assets/fonts/pacifico/pacifico.css",
+                            "dist/core/assets/fonts/material-design-icons/material-icons.css"
+                        ],
+                        "scripts": []
+                    },
+                    "configurations": {
+                        "production": {
+                            "budgets": [
+                                {
+                                    "type": "initial",
+                                    "maximumWarning": "5mb",
+                                    "maximumError": "10mb"
```

Comment:
holy moly, isn't this quite large?

Reply:
these are your values. i just copied them from the other projects